BloodNight-core/src/main/java/de/eldoria/bloodnight/specialmobs/SpecialMobUtil.java

```diff
@@ -184,7 +185,8 @@ public static <T extends Entity> T spawnAndMount(Entity carrier, EntityType pass
     public static <T extends Entity> T spawnAndMount(EntityType carrierType, Entity rider) {
         T carrier = spawnAndTagEntity(rider.getLocation(), carrierType);
         assert carrier == null;
-        tagExtension(carrier, rider);
+        Optional<String> mobName = getSpecialMobType(carrier);
```

Comment:
In that case the rider defines the mob type. The carrier is a new mob.
```suggestion
        Optional<String> mobName = getSpecialMobType(rider);
```